### DISCUSSION TITLE
Organization新着一覧が怪しいサイトで埋め尽くされている

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#387] [要望・アイデア・不具合]
タイトルの通りに[Organization新着一覧](https://qiita.com/organizations)に連日、海外勢から明らかに本来の利用目的でない、Organizationが登録されて怪しい薬などの広告に利用されているように見受けられます。

以下本日、見つけたキャプチャですが、上から下まで全て埋め尽くされています。
中には同じメールアドレスで複数登録されているのもあります。

![スクリーンショット 2022-12-22 22 37 30](https://user-images.githubusercontent.com/38848514/209146528-1e5e0afe-0a4c-4d02-a5ef-62063d9b4d09.png)
![スクリーンショット 2022-12-22 22 37 37](https://user-images.githubusercontent.com/38848514/209146535-fffd26f2-ed02-4fc4-aaae-380f0f330626.png)

気をつけないといけないのが、これらの中にフィッシング サイトのようなものが存在する場合、犯罪に利用される可能性も出てきます。

つきましては、Organizationを申請制度にするとか、運営側で見つけたら削除するなど何らか対応及び対策をした方が良いのではないでしょうか。目につくだけでも気持ち悪いので、早めに対応いただけると幸いです。

以上です。よろしくお願いします。

**Comment by tanaka-asami415** [comment]
@Brutus03 
ご報告ありがとうございます🙏
OrganizationはQiita上での活動が一定の基準に達していないと、作成できないように制限をかけているのですが
その判定が正しくできていない可能性があるため、調査いたします！🙇

**Reply by tanaka-asami415** [reply]
@Brutus03 
調査した結果、判定が正しく動作しておりませんでした。対応完了しております！
ご報告いただきありがとうございました🙇

**Reply by Brutus03** [reply]
ご対応ありがとうございます。
本件はクローズで大丈夫です。

以上です。
